enterprise_access/apps/subsidy_request/models.py

```diff
@@ -0,0 +1,162 @@
+""" Models for subsidy_request. """
+
+from uuid import uuid4
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+from enterprise_access.apps.subsidy_request.constants import SubsidyRequestStates
+
+
+class SubsidyRequest(TimeStampedModel):
+    """
+    Stores information related to a request for a subsidy (license or coupon).
+
+    .. no_pii: This model has no PII
+    """
+
+    uuid = models.UUIDField(
+        primary_key=True,
+        default=uuid4,
+        editable=False,
+        unique=True,
+    )
+
+    lms_user_id = models.IntegerField()
+
+    course_id = models.CharField(
+        null=True,
+        blank=True,
+        max_length=128
+    )
+
+    customer_uuid = models.UUIDField()
+
+    state = models.CharField(
+        max_length=25,
+        blank=False,
+        null=False,
+        choices=SubsidyRequestStates.CHOICES,
+        default=SubsidyRequestStates.PENDING_REVIEW
+    )
+
+    reviewed_at = models.DateTimeField(
+        null=True,
+        blank=True
+    )
+
+    reviewer_id = models.IntegerField(
+        null=True,
+        blank=True
+    )
```

Comment:
Is this the lms user id of the reviewer, or a FK to `core.User`?

Reply:
this is the lms user id of the reviewer, I should maybe change lms_user_id -> requester_lms_user_id and this to reviewer_lms_user_id?

Reply:
yeah, or take the position of - everything related to the reviewer has `reviewer` as a prefix, otherwise the prefix is the empty string.  ex.
`lms_user_id` - The requesting learner's LMS user id
`reviewer_lms_user_id` - The reviewing admin's LMS user id